app/routes.js

```diff
@@ -10,6 +10,7 @@ module.exports = function(app = choo()) {
   app.route('/legal', body(require('./ui/legal')));
   app.route('/error', body(require('./ui/error')));
   app.route('/blank', body(require('./ui/blank')));
+  // app.route('/qr', body(require('./ui/displayQR')))
```

Comment:
Better to remove this commented out route if its not in use

Reply:
Good point! Fixed that now